### DISCUSSION TITLE
Flood crash randomly

### DISCUSSION THREAD

**Discussion by mouzzampk2014-zz** [jesec/flood#354] [Q&A]
Hi, I see these but not all the time
```
today at 14:27:06 Caught internal_error: FileList::mark_completed(...) received a chunk that has already been finished. [#8568A992E9F4B02307ACAE3C80906BC9709CC418]
today at 14:27:06 Stack dump not enabled.
```
I am not sure what's causing this but most recently I stopped and resume the torrent and its crashed.

Thanks

**Comment by jesec** [comment]
What rTorrent version do you use?

**Reply by mouzzampk2014-zz** [reply]
rtorrent/0.9.8/0.13.8 and Flood 4.6.0.

**Reply by jesec** [reply]
Which distribution of rTorrent?

**Reply by mouzzampk2014-zz** [reply]
I am using `hotio/rflood` image

**Reply by jesec** [reply]
I see. I will take a look.

**Reply by mouzzampk2014-zz** [reply]
Just to give you heads up. Since yesterday I cannot add any torrents but before when it happened I was able to add the torrents after few hours. 

Thanks

**Reply by jesec** [reply]
That's very weird.

**Reply by mouzzampk2014-zz** [reply]
Can we close this one please #352 its related to this issue.

**Reply by jesec** [reply]
I think we can close that thread instead. 

I just did a quick test, and unfortunately I can not reproduce the issue.

**Reply by mouzzampk2014-zz** [reply]
Yes thats fine because I evenly don't know why its happening :)

**Reply by jesec** [reply]
This appears to be a longstanding and rare rTorrent issue: rakshasa/rtorrent#313, rakshasa/rtorrent#503, rakshasa/rtorrent#565, binhex/arch-rtorrentvpn#52. 

Can you provide a minimum method to reproduce the issue? 

I still can't reproduce it, which makes it hard to work on it.

**Reply by mouzzampk2014-zz** [reply]
Yesterday I added a torrent as `Use as Base Path` and `Completed`. Once torrent was added I started the torrent and torrent was was already 99% completed because I had files in the system already but remaining 1% need downloading. After waiting 5 minutes I have still 1% left so I thought why not pause the torrent and start the torrent again and once I did that. Whole rflood crashed and restarted again and since than I couldn't add any files in any way and keep getting Failed to add 1 torrent or 5 torrents. 

I am not sure if that helps.

Thanks

**Reply by mouzzampk2014-zz** [reply]
Hi @jesec so I restarted the container but still unable to add torrent so I decided to restart the whole system and now I can add torrents. I thought I should update you. 

Thanks

**Reply by jesec** [reply]
Hmm.... Maybe your system runs out of resources (disk, memory, etc.), or you somehow set a limit on the containers?

**Reply by mouzzampk2014-zz** [reply]
No clue but I have enough resources where I am running 32GB RAM and i7 4790k. Lets see if it happens again and I will update this thread. Ta
